test/powershell/Language/Operators/ReplaceOperator.Tests.ps1

```diff
@@ -0,0 +1,67 @@
+Describe "Split Operator" -Tags CI {
```

Comment:
Typo.

Reply:
Thanks @iSazonov, fixed in commit aa54fe56a9efb2c0c49b551599ec193d67c47e36